### DISCUSSION TITLE
谁能告诉我这是什么意思？

### DISCUSSION THREAD

**Discussion by huhujieshuo** [Meloong-Git/PCL#1922] [提问] [answered]
![image](https://github.com/Hex-Dragon/PCL2/assets/132453725/c4b8629a-6e0f-4682-b6f9-0c6f76130220)

**Comment by iPodtouch6557** [comment, marked as answer]
这个帮助库指的是 [ PCL2Help ]( https://github.com/LTcatt/PCL2Help )
当然了这个存储库也行的

**Comment by Chiloven945** [comment]
pull request你可以理解为提交申请以加你自己的东西(代码)的意思
